rpmlint/checks/UpdateAlternativesCheck.py

```diff
@@ -0,0 +1,79 @@
+from os.path import basename
+import stat
+
+import rpm
+from rpmlint.checks.AbstractCheck import AbstractCheck
+from rpmlint.pkg import b2s
+
+
+class UpdateAlternativesCheck(AbstractCheck):
+    """
+    Check for compliance with SUSE's update-alternatives guidelines:
+      http://en.opensuse.org/openSUSE:Packaging_Multiple_Version_guidelines
```

Comment:
Can we make this comment more generic?

Reply:
Adjusted it somehow, still i think we should rather point to the docs than describe it again in here? Or should i copy the wiki content there?

Reply:
If it's not too much content, perhaps inlining it makes sense? I'm not sure.

Reply:
added there.